tests/jerry/es.next/nullish-coalescing.js

```diff
@@ -0,0 +1,67 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//basic valid cases with logical operator
+assert((1 ?? 2) == 1)
+assert((0 ?? 2) == 0)
+assert(null ?? 2 == 2)
+assert(null ?? undefined == undefined)
+assert(null ?? undefined ?? 2 == 2)
+assert(null ?? undefined ?? null ?? 10  == 10)
+assert(null ?? (undefined || null) ?? 10  == 10)
+assert(null ?? (undefined && null) ?? 10  == 10)
+assert((null ?? true) && (true ?? null) == true)
+
+//cannot evulate the right expression if left is not null or undefined
+function poison()
+{
```

Comment:
Move up this brace after `()`.

Reply:
This request is still valid.

Reply:
Still.

Reply:
I completely misunderstood your request. Sorry for the inconvenience